src/interface/src/app/shared/form-message-box/form-message-box.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit } from '@angular/core';
+//import { FormMessageType } from '../../types/data.types';
+
+@Component({
+  selector: 'app-form-message',
+  templateUrl: './form-message-box.component.html',
+  styleUrls: ['./form-message-box.component.scss'],
+})
+export class FormMessageBoxComponent implements OnInit {
+  @Input() title!: string;
+  @Input() message?: string;
+  @Input() showMessage: boolean = false;
+  @Input() messageType: string = '';
+
+  constructor() {}
+
+  getMessageIcon(): string {
+    switch (this.messageType) {
+      case 'ALERT':
+        return 'assets/svg/icons/exclamation-square-fill.svg';
+      case 'ERROR':
+        return 'assets/svg/icons/warning-icon-fill.svg';
+      case 'SUCCESS':
+        return 'assets/svg/icons/checkmark-round-fill.svg';
+      default: // alert is default
+        return 'assets/svg/icons/exclamation-square-fill.svg';
+    }
+  }
+
+  getBoxClass(): string {
```

Comment:
I think if you define a type you can remove this and just use it directly on the html. If not, you can also use a record as the example above